### PR TITLE
Great Renaming, Take 3

### PR DESCRIPTION
**Pull request by vampcat**
Basically #254, with merge conflicts fixed.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Uh oh, something went wrong with the build. Need to check on that

**Comment by Adrijaned** [comment]
This is already a third PR of this renaming. Before, there was #254 and #253.

**Comment by vampcat** [comment]
There ya go, fixed it.

Now can you please provide some *meaningful* review? :P

File: engine/src/main/java/org/destinationsol/game/chunk/ChunkFiller.java [hunk 1]
```diff
@@ -295,31 +296,35 @@ private void fillJunk(SolGame game, RemoveController remover, SpaceEnvConfig con
...
-            Vector2 junkPos = getRndPos(chCenter);
+            // Select a random position in the chunk centered around chunkCenter, relative to the entire map.
+            Vector2 junkPos = getRndPos(chunkCenter);
 
             // Select a random junk atlas
+<<<<<<< HEAD
             TextureAtlas.AtlasRegion tex = new TextureAtlas.AtlasRegion(SolRandom.randomElement(conf.junkTexs));
+=======
+            TextureAtlas.AtlasRegion tex = new TextureAtlas.AtlasRegion(SolMath.elemRnd(conf.junkTextures));
+>>>>>>> 4df19c3... Some renaming and refactoring of basically everything, yet another round
```
Comment (Adrijaned) [inline engine/src/main/java/org/destinationsol/game/chunk/ChunkFiller.java]: WTH, vampy... You gotta try harder.

Comment (vampcat) [reply engine/src/main/java/org/destinationsol/game/chunk/ChunkFiller.java]: To be entirely fair, I did not.

**Comment by vampcat** [comment]
Also, I *did* try running it this time :P

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by Adrijaned** [comment]
Seems fine. Exiting and rejoining game while transcendentheroing through sun kills you, but that's Ghosts's issue, not mine.

**Pull request merged**